### DISCUSSION TITLE
Merge local .env when downloading Development Environment Variables

### DISCUSSION THREAD

**Discussion by wyeo** [vercel/vercel#4072] [General] [answered]
That would be good

**Comment by coetry** [comment, marked as answer]
We want to have development env vars synced in the cloud so that it can be distributed to collaborators in a secure way without having to check them into source control.

**Reply by styfle** [reply]
Put another way, you should not be editing your `.env` file manually.

Instead use `now env add` and then `now env pull`. 

Then everyone working on the same project can do `git pull` followed by `now env pull` to stay in sync with the latest changes.
